SJMaster/urls/__init__.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+from django.urls import path
+from .student_urls import urlpatterns as student_urlpatterns
+from .recruiter_urls import urlpatterns as recruiter_urlpatterns
+from .login_urls import urlpatterns as login_urlpatterns
+from .jobboard_urls import urlpatterns as jobboard_urlpatterns
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+]
+
+urlpatterns += student_urlpatterns
+urlpatterns += recruiter_urlpatterns
+urlpatterns += login_urlpatterns
+urlpatterns += jobboard_urlpatterns
```

Comment:
`django.urls` has an `include` function just for this.

Reply:
ok

Reply:
done